src/applications/hca/selectors.js

```diff
@@ -25,8 +25,10 @@ export const isUserLOA3 = state =>
   !hasServerError(state) &&
   !noESRRecordFound(state) &&
   isLOA3(state);
+export const isLoggedOut = state =>
```

Comment:
`isLoggedOut` is now a much simpler selector the really does only check if the user is logged out.